.github/composite_actions/setup_chromedriver/dist/index.mjs

```diff
@@ -0,0 +1,6889 @@
+import { createRequire as __WEBPACK_EXTERNAL_createRequire } from "module";
```

Comment:
Where is this code generated or pulled from? Apologies if I am missing something obvious.

Reply:
I see. Could it be generated prior to use rather than committed?